ansible/roles/test/files/ptftests/arista.py

```diff
@@ -373,7 +373,13 @@ def change_bgp_neigh_state(self, asn, is_up=True):
         state = ['shut', 'no shut']
         self.do_cmd('configure')
         self.do_cmd('router bgp %s' % asn)
-        self.do_cmd('%s' % state[is_up])
+        if is_up == True:
+            self.do_cmd('%s' % state[is_up])
+        else:
+            # shutdown BGP will pop confirm message, the message is 
+            # "You are attempting to shutdown BGP. Are you sure you want to shutdown? [confirm]"
+            self.do_cmd('%s' % state[is_up], prompt = '[confirm]')
+            self.do_cmd('y')
```

Comment:
does it also prompt, what if it does not prompt? i believe the existing works for some platform. your changes seem to break some of the existing vm. can you make it backward compatible?

Reply:
commit updated, add to check the veos version.